MFlix/MFlix/Scenes/Search/SearchUseCase.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SearchUseCase.swift
+//  MFlix
+//
+//  Created by Viet Anh on 5/6/20.
+//  Copyright © 2020 VietAnh. All rights reserved.
+//
+
+protocol SearchUseCaseType {
+    func searchListMovie(query: String) -> Observable<PagingInfo<Movie>>
+    func loadMoreMovie(query: String, page: Int) -> Observable<PagingInfo<Movie>>
+}
+
+struct SearchUseCase: SearchUseCaseType {
+
+    private let repository = SearchRepository()
+    
+    func searchListMovie(query: String) -> Observable<PagingInfo<Movie>> {
+        return loadMoreMovie(query: query, page: 1)
+    }
+    
+    func loadMoreMovie(query: String, page: Int) -> Observable<PagingInfo<Movie>> {
+        let text = query.isEmpty ? "abcdefghjklm" : query
```

Comment:
```suggestion
        let text = query.isEmpty ? "abcdefghjklm" : query
```
abcdefghjklm gì đây e ?